src/main/java/org/ideoholic/curium/model/job/dto/ReportResponseDto.java

```diff
@@ -15,10 +17,12 @@ public class ReportResponseDto {
 	
 	private String statusSelected;
 	private String staffSelected;
+	private String studentselected;
 	private List<JobQuery> JobQueryList;
 	private String transactionFromDateSelected;
 	private String transactionToDateSelected;
 	private String querycompleted;
 	private boolean success;
+	private List<Task> jobQueriList;
```

Comment:
private List<JobQuery> JobQueryList;
private List<Task> jobQueriList;
There are two list of same name therefore change in spelling otherwise it will give error